docs/corpora_schema.md

```diff
@@ -11,19 +11,20 @@ Date Last Modified: 2020-12-11
 
 ## Background
 
-cellxgene aims to support the publication and sharing of single-cell datasets as well as the construction of a data corpus that facilitates data integration
-across multiple tissues and experiments. Achieving the latter goal requires some harmonization of metadata and features in the cellxgene Data Portal. But if
-that harmonization is too onerous, it will burden the goal of rapid data sharing.
+cellxgene aims to support the publication, sharing, and exploration of single-cell datasets.
+Building on those published datasets, cellxgene seeks to create references of the phenotypes and composition of cells that make up human tissues.
+Creating references from multiple datasets requires some harmonization of metadata and features in the cellxgene Data Portal. But if that harmonization is too onerous, it
+will burden the goal of rapid data sharing.
 
-We balance these two goals by requiring datasets hosted in the cellxgene Data Portal to follow a small schema with only a few required fields. These fields
+cellxgene balances publishing and reference creation needs by requiring datasets hosted in the cellxgene Data Portal to follow a small schema with only a few required fields. These fields
 are expected to be very useful for data integration and also simply and readily available from data submitters.
 
-Note that the requirements in the schema are just the minimum required information. We expect that datasets will have additional metadata, and this
-will be preserved in datasets submitted to the Data Portal.
+Note that the requirements in the schema are just the minimum required information. Datasets often have additional metadata, which
+is preserved in datasets submitted to the Data Portal.
 
 ## Curation and Validation
 
-When a submitter is preparing a dataset for the cellxgene Data Portal, they will be able to use a command line tool to apply changes to their dataset so that it
+When a submitter is preparing a dataset for the cellxgene Data Portal, they can use the `cellxgene-schema apply` command line tool to apply changes to their dataset so that it
```

Comment:
Mmm. Isn't it a two step process and wouldn't it be better to simply reference the CLI documentation:

You can use the config file to augment a dataset with  the schema using:

```
cellxgene-schema apply --source-h5ad original.h5ad --remix-config config.yml --output-filename remixed.h5ad
```

And then verify that the schema was properly added with:

```
cellxgene-schema validate remixed.h5ad
```

Reply:
I removed this from the document when refactoring in #52 to remove the encoding. It belongs in `schema_guide.md` (or the tutorial that will replace it). 

I also added an explanation of when to read this document so that users seeking curation advice are quickly sent to the schema_guide.